client/my-sites/store/index.js

```diff
@@ -157,41 +37,6 @@ function redirectIfWooCommerceNotInstalled( context, next ) {
 }
 
 export default async function () {
-	// Without site param
-	page( '/store', siteSelection, sites, makeLayout, clientRender );
-
-	// Dashboard
-	page(
-		'/store/:site',
-		siteSelection,
-		redirectIfWooCommerceNotInstalled,
-		( context, next ) => {
-			const component = createElement( Dashboard, {
-				params: context.params,
-			} );
-			const appProps = {
-				analyticsPath: getAnalyticsPath( '/store/:site', context.params ),
-				analyticsTitle: 'Store > Dashboard',
-			};
-
-			context.primary = createElement( App, appProps, component );
-
-			context.secondary = createElement( StoreSidebar, { path: context.path } );
-
-			next();
-		},
-		makeLayout,
-		clientRender
-	);
-
-	// Redirect all other pages to the dashboard, to show the Store removal notice
-	getStorePages().forEach( function ( storePage ) {
-		page( storePage.path, redirectIfWooCommerceNotInstalled, ( context ) => {
-			context.store.dispatch( bumpStat( 'calypso_store_post_sunset', storePage.statName ) );
```

Comment:
Just verified the `calypso_store_post_sunset` stats are too low to be worth keeping the redirects.